### PR TITLE
Allow reconciliation with context

### PR DESCRIPTION
**Pull request by pierventre**

### CODE REVIEW

**Review by eroshiva — APPROVED** [review]

**Comment by dafATonf** [comment]
test this please

**Comment by SeanCondon** [comment]
retest this please

**Comment by SeanCondon** [comment]
I've tried running this [manually](https://jenkins.aetherproject.org/job/verify_onos-lib-go_make-test/282/console) in Jenkins 
Fails with error 
```
07:37:54 Error loading key "/home/jenkins/workspace/verify_onos-lib-go_make-test@tmp/private_key_6482476696601047681.key": invalid format
```

**Comment by pierventre** [comment]
any update ?

**Comment by SeanCondon** [comment]
> any update ?

Nothing unfortunately. I missed the Aether TST on Tuesday, but I had asked Scott to bring it up. I've pinged @dafATonf several times on Slack but no response.

**Comment by pierventre** [comment]
no need to merge this now. still working through the details

**Comment by gab-arrobo** [comment]
ok to test

**Comment by gab-arrobo** [comment]
@pierventre, is this PR ready for review? or are you planning to make more changes?

File: pkg/controller/v2/controller.go [hunk 1]
```diff
@@ -1,11 +1,12 @@
 // SPDX-FileCopyrightText: 2023-present Intel Corporation
-//
+//
```
Comment (gab-arrobo) [inline pkg/controller/v2/controller.go:2]: Jekins is failing due to formatting in pkg/controller/v2/controller.go
```suggestion
//
```

**Review by SeanCondon — APPROVED** [review]

**Review by raphaelvrosa — APPROVED** [review]

**Comment by SeanCondon** [comment]
Thanks for approval @raphaelvrosa - can I merge?

**Comment by pierventre** [comment]
> Thanks for approval @raphaelvrosa - can I merge?

if @kuujo is ok with this, yes!

**Pull request closed without merging**